examples/taskflow/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2016-2020, Rahul Sheth, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.2)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-taskflow)
+
+# DOCUMENTATION_START {
+hunter_add_package(taskflow)
+find_package(Taskflow CONFIG REQUIRED NO_CMAKE_PACKAGE_REGISTRY)
```

Comment:
This is probably fine, we just need to keep it in the example so it shows up in the docs (which it will now).